### PR TITLE
Bump `@mui/monorepo`

### PR DESCRIPTION
**Pull request by DanailH**
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/mui-x/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: docs/pages/x/api/charts/charts-axis.json [hunk 1]
```diff
@@ -37,7 +37,24 @@
     "import { ChartsAxis } from '@mui/x-charts/ChartsAxis';",
     "import { ChartsAxis } from '@mui/x-charts';"
   ],
-  "styles": { "classes": [], "globalClasses": {}, "name": "MuiChartsAxis" },
+  "styles": {
+    "classes": [
```
Comment (LukasTy) [inline docs/pages/x/api/charts/charts-axis.json:41]: This change looks strange to me. Doesn't looks like this component actually has these slots. 🙈 
@alexfauquette could you take a look?

Comment (LukasTy) [reply docs/pages/x/api/charts/charts-axis.json:41]: Clarification, the generated change seems correct given the existing themeAugmentation declaration, but the registered slots do not seem to be consumed and/or applied anywhere... 🤷

Comment (alexfauquette) [reply docs/pages/x/api/charts/charts-axis.json:41]: The classes exist, but the `styleOverride` don't if that's what you mean. But it's a problem on `x-charts` side

Comment (LukasTy) [reply docs/pages/x/api/charts/charts-axis.json:41]: I've looked more deeply into this matter and my initial assumption was incorrect.
This is actually a needed fix.
But I believe I see a different problem regarding `MuiChartsXAxis` and `MuiChartsYAxis`.

**Review by LukasTy — APPROVED** [review]
LGTM. 👌

File: docs/pages/x/api/charts/charts-axis.json (hunk 1 above)
Comment (LukasTy) [reply docs/pages/x/api/charts/charts-axis.json:41]: Fine from my PoV. I've prepared a PR removing the not working slots: https://github.com/mui/mui-x/pull/9996

**Pull request merged**